src/pages/rewards/RewardsPage.tsx

```diff
@@ -1,134 +1,49 @@
-                {stringGetter({ key: STRING_KEYS.MIGRATE })}
-              </Styled.MigrateTitle>
-              <Styled.Description>
-                {stringGetter({ key: STRING_KEYS.MIGRATE_DESCRIPTION })}
-                <Link href={MIGRATE_HELP_URL}>
-                  {stringGetter({ key: STRING_KEYS.LEARN_MORE })} →
-                </Link>
-              </Styled.Description>
-            </div>
-            <Styled.MigrateAction>
-              <div>
-                <div>{stringGetter({ key: STRING_KEYS.AVAILABLE_TO_MIGRATE })}</div>
-                <div>
-                  <Styled.Token type={OutputType.Asset} value={tokenBalance} />
-                </div>
-              </div>
-              <Button
-                action={tokenBalance ? ButtonAction.Primary : ButtonAction.Base}
-                // type={ButtonType.Link}
-                // href={BRIDGE_URL}
-                disabled
-              >
-                {tokenBalance
-                  ? `${stringGetter({ key: STRING_KEYS.MIGRATE_NOW })} →`
-                  : stringGetter({ key: STRING_KEYS.NO_TOKENS_TO_MIGRATE })}
-              </Button>
-            </Styled.MigrateAction>
-          </Styled.TwoItemRow>
-        </Styled.Migrate>
-      ) : (
-        <Styled.MobileMigrateCard
-          slotHeader={
-            <Styled.MobileMigrateHeader>
-              <h3>{stringGetter({ key: STRING_KEYS.MIGRATE })}</h3>
-              <VerticalSeparator />
-              <span>
-                {stringGetter({
-                  key: STRING_KEYS.FROM_TO,
-                  params: { FROM: <strong>Ethereum</strong>, TO: <strong>dYdX Chain</strong> },
-                })}
-              </span>
-            </Styled.MobileMigrateHeader>
-          }
-        >
-          <Styled.WithReceipt
-            slotReceipt={
-              <Styled.Details
-                items={[
-                  {
-                    key: 'available-to-migrate',
-                    label: stringGetter({ key: STRING_KEYS.AVAILABLE_TO_MIGRATE }),
-                    value: <Output type={OutputType.Asset} value={tokenBalance} />,
-                  },
-                ]}
-              />
-            }
-          >
-            <Button
-              action={tokenBalance ? ButtonAction.Primary : ButtonAction.Base}
-              size={ButtonSize.Medium}
-              disabled
-            >
-              {tokenBalance
-                ? `${stringGetter({ key: STRING_KEYS.MIGRATE_NOW })} →`
-                : stringGetter({ key: STRING_KEYS.NO_TOKENS_TO_MIGRATE })}
-            </Button>
-          </Styled.WithReceipt>
-          <Styled.LearnMore>
-            {stringGetter({ key: STRING_KEYS.WANT_TO_LEARN })}
-            <Link href={MIGRATE_HELP_URL}>{stringGetter({ key: STRING_KEYS.CLICK_HERE })}</Link>
-          </Styled.LearnMore>
-        </Styled.MobileMigrateCard>
-      )} */}
-
+      {import.meta.env.VITE_V3_TOKEN_ADDRESS && <MigratePanel />}
```

Comment:
Should this be added to `.env.local` as well?

Reply:
already is!